Automattic/jetpack
# Boost: Implement "Clear Cache" UI

Fixes #35783

## Proposed changes:

* Clear page cache when clicking the "Clear Cache" button;
* Disable "Clear Cache" button during cleanup;
* Update the Page Cache summary during cleanup;
* Add snackbar message after cleanup is done.

Currently, when cleaning the cache, the "clean cache" PHP side of things returns an error that the cache directory doesn't exist. This is why the snackbar message shows on success and error.

### Other information:


## Jetpack product discussion



pc9hqz-2rA-p2

pc9hqz-2vF-p2

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Make sure you have some cached pages;
* Click the "Clear Cache" button and make sure cache is deleted (and that the message shows up).
